src/App.js

```diff
@@ -1,8 +1,20 @@
 import React from "react"
+import { Route, BrowserRouter, Routes } from "react-router-dom";
+import Email from './components/pages/emailExist'
+import Sent from './components/pages/emailSent'
+import Main from './components/pages/resetPassword';
 import Navbar from "./components/Navbar/Navbar";
 function App() {
     return (<div>
-        <Navbar />
+      <Navbar/>
+        <BrowserRouter>
+        <Routes>
+          <Route exact path="/" element={<Main/>}></Route>
```

Comment:
On this main Route, you should remove your component. And leave it to your route only.

Reply:
You can also use Routes specific to Resetting passwords. Like,
On first page: /reset-password
On second page: /reset-password/comfirm
On third page: /reset-password/email-sent

This is an example, but something like this would be more understandable and people would know related routes.